rust/noosphere-core/src/data/bundle.rs

```diff
@@ -182,49 +179,26 @@ impl TryBundle for BodyChunkIpld {
 impl<K, V> TryBundle for ChangelogIpld<MapOperation<K, V>>
 where
     K: VersionedMapKey,
-    V: VersionedMapValue,
+    V: VersionedMapValue + TryBundle,
 {
     async fn try_extend_bundle_with_cid<S: BlockStore>(
```

Comment:
consider: removal of `try_` prefix, like the other `noosphere` (IIRC?) changes

Reply:
`try_` removed